docs/dev/table/tableApi.md

```diff
@@ -188,7 +188,50 @@ Table result = orders.select("*");
 {% endhighlight %}
       </td>
     </tr>
-
+  <tr>
+          <td>
+            <strong>AddColumns</strong><br>
```

Comment:
Thanks for reminding those case:
 When: replaceIfExist = false(default)
 1. It will throw an exception if the added fields already exist.
 2. It will throw an exception if adding columns with the same name
 3. It will throw an exception when you add aggregation as a new column
When: replaceIfExist = true
 4.  It will throw an exception if replace the existing column but do not with the alias.
 5.  If the added fields have duplicate field name, then the last one is valid.